mlflow/java/spark/pom.xml

```diff
@@ -137,9 +170,4 @@
       </plugin>
     </plugins>
   </build>
-  <profiles>
-    <profile>
-      <id>do-sign</id>
```

Comment:
Can we remove this ?

Reply:
Yup, but not super sure what it does?. I saw [your PR](https://github.com/mlflow/mlflow/pull/4181) to adding this profile, but would removing it change the process of publishing artifacts to the maven central?

Reply:
It is for signing jars when publishing, we define `do-sign` profile in parent pom, so when running mvn with this profile enabled, the generated jars will be signed. But I think sub-project should inherit the `do-sign` profile for supporting signing.